packages/core/saved-objects/core-saved-objects-migration-server-internal/src/actions/read_with_pit.ts

```diff
@@ -93,5 +103,22 @@ export const readWithPit =
           totalHits,
         });
       })
+      .catch((e) => {
+        if (
+          e instanceof EsErrors.RequestAbortedError &&
+          e.message.match(/The content length \(\d+\) is bigger than the maximum/) != null
```

Comment:
nit: `.test` might be more performant because it doesn't need to extract the groups. I don't have numbers that back this assumption though.

```suggestion
          /The content length \(\d+\) is bigger than the maximum/.test(e.message)
```